docs/tutorials/01_KalmanFilterTutorial.py

```diff
@@ -170,6 +177,8 @@
 # ground truths, measurements and tracks in a consistent format. It can be accessed by importing
 # the class :class:`Plotter` from Stone Soup as below.
 
+# Note that the mapping argument is [0, 2] because those are the x and y position indices from our state vector.
```

Comment:
This line is part of the code cell. Did you want this? If so, could it be split in to 2 lines so that we don't need a slider to read the whole sentence?
If not, you just need to put a `#` on the blank line above.

Reply:
Added `#` to join it to the text cell.